<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QWidget" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="visualTab">
      <attribute name="title">
       <string>Visual</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="meshGroupBox">
             <property name="title">
              <string>Mesh</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0" alignment="Qt::AlignRight">
                 <widget class="QLabel" name="linearDeflectionLabel">
                  <property name="text">
                   <string>&amp;Linear Deflection</string>
                  </property>
                  <property name="buddy">
                   <cstring>linearDeflectionEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="angularDeflectionEdit"/>
                </item>
                <item row="1" column="0" alignment="Qt::AlignRight">
                 <widget class="QLabel" name="angularDeflectionLabel">
                  <property name="text">
                   <string>&amp;Angular Deflection</string>
                  </property>
                  <property name="buddy">
                   <cstring>angularDeflectionEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="linearDeflectionEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="draggerGroupBox">
             <property name="title">
              <string>Dragger</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QGridLayout" name="draggerGroup">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0" alignment="Qt::AlignRight">
                   <widget class="QLabel" name="linearIncrementLabel">
                    <property name="text">
                     <string>Linear Increment</string>
                    </property>
                    <property name="buddy">
                     <cstring>linearIncrementEdit</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="linearIncrementEdit"/>
                  </item>
                  <item row="1" column="0" alignment="Qt::AlignRight">
                   <widget class="QLabel" name="angularIncrementLabel">
                    <property name="text">
                     <string>Angular Increment</string>
                    </property>
                    <property name="buddy">
                     <cstring>angularIncrementEdit</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="angularIncrementEdit"/>
                  </item>
                  <item row="2" column="0" alignment="Qt::AlignRight">
                   <widget class="QLabel" name="updateOnFinishLabel">
                    <property name="text">
                     <string>Update On Finish</string>
                    </property>
                    <property name="buddy">
                     <cstring>updateOnFinishCombo</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QComboBox" name="updateOnFinishCombo">
                    <item>
                     <property name="text">
                      <string>True</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>False</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="interactiveParameterGroupBox">
             <property name="title">
              <string>Interactive Parameter</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="InteractiveParameterLayout">
                <item row="0" column="0" alignment="Qt::AlignRight">
                 <widget class="QLabel" name="characterSizeLabel">
                  <property name="text">
                   <string>Character Size</string>
                  </property>
                  <property name="buddy">
                   <cstring>characterSizeEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="arrowWidthEdit"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="arrowHeightEdit"/>
                </item>
                <item row="2" column="0" alignment="Qt::AlignRight">
                 <widget class="QLabel" name="arrowHeightLabel">
                  <property name="text">
                   <string>Arrow Height</string>
                  </property>
                  <property name="buddy">
                   <cstring>arrowHeightEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="characterSizeEdit"/>
                </item>
                <item row="1" column="0" alignment="Qt::AlignRight">
                 <widget class="QLabel" name="arrowWidthLabel">
                  <property name="text">
                   <string>Arrow Width</string>
                  </property>
                  <property name="buddy">
                   <cstring>arrowWidthEdit</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>verticalSpacer</zorder>
     </widget>
     <widget class="QWidget" name="projectTab">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="basePathLabel">
           <property name="text">
            <string>Base Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="basePathEdit">
           <property name="whatsThis">
            <string>Default path for projects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="basePathButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gitGroupBox">
         <property name="title">
          <string>Git</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="gitNameLabel">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="gitNameEdit">
              <property name="whatsThis">
               <string>Name used in git commits</string>
              </property>
              <property name="text">
               <string>Your Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="gitEmailLabel">
              <property name="text">
               <string>EMail</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="gitEmailEdit">
              <property name="whatsThis">
               <string>EMail used in git commits</string>
              </property>
              <property name="text">
               <string>Your EMail</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>252</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>linearDeflectionEdit</tabstop>
  <tabstop>angularDeflectionEdit</tabstop>
  <tabstop>linearIncrementEdit</tabstop>
  <tabstop>angularIncrementEdit</tabstop>
  <tabstop>updateOnFinishCombo</tabstop>
  <tabstop>characterSizeEdit</tabstop>
  <tabstop>arrowWidthEdit</tabstop>
  <tabstop>arrowHeightEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
